app/src/main/res/layout/fragment_stock_exchange.xml

```diff
@@ -236,4 +237,39 @@
         android:orientation="vertical"
         app:layout_constraintGuide_percent="0.6" />
 
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="left"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toTopOf="@+id/buyExchangeButton"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.826"
+        app:layout_constraintStart_toStartOf="@+id/stocks_exchange_input"
+        app:layout_constraintTop_toBottomOf="@+id/stocksInExchange_textView">
+
+        <Button
+            android:id="@+id/stockIncrementButton"
+            android:layout_width="35dp"
+            android:layout_height="35dp"
+            android:layout_marginTop="40dp"
+            android:background="@drawable/increment_arrow"
```

Comment:
Instead of background, just put text, like **+1** or **+5**
Because arrows don't show how much it is going to add

Reply:
yeah i have changed this to +5 and -5